lib/model/model.go

```diff
@@ -1057,9 +1153,40 @@ func (m *model) ClusterConfig(deviceID protocol.DeviceID, cm protocol.ClusterCon
 		if !ok {
 			// Shouldn't happen because !cfg.Paused, but might happen
 			// if the folder is about to be unpaused, but not yet.
+			l.Debugln("ccH: no fset", folder.ID)
 			continue
 		}
 
+		ccDeviceRemote, hasDevice := ccDevicesRemote[folder.ID]
+		ccDeviceLocal, hasDeviceLocal := ccDevicesLocal[folder.ID]
+
+		if err := m.ccCheckEncryption(cfg, folderDevice, ccDeviceRemote, ccDeviceLocal, hasDevice, hasDeviceLocal, deviceCfg.Untrusted); err != nil {
+			sameError := false
+			if devs, ok := m.folderEncFailures[folder.ID]; ok {
+				sameError = devs[deviceID] == err
+			} else {
+				m.folderEncFailures[folder.ID] = make(map[protocol.DeviceID]error)
+			}
+			m.folderEncFailures[folder.ID][deviceID] = err
+			msg := fmt.Sprintf("Failure checking encryption consistency with device %v for folder %v: %v", deviceID, cfg.Description(), err)
```

Comment:
What is encryption consistency? and how do I restore it?

I think this talks about disagreement on encryption passwords not matching?

Reply:
`err` has the specifics. It can mean a few things, from not matching PWs, to remote sending us encrypted stuff while we think we should get plain things, or the other way around, or ...  
Maybe just: "Incorrect encryption configuration for folder %v with device %v"?

Reply:
I guess if it can mean many things we should report each fault separately to tell the user where the problem lies. Saying that there is some undefined problem which might be one from this long list is not super helpful.

Reply:
We are not logging an unspecific error or giving a list, but e.g. `Failure checking encryption consistency with device *somedevice* for folder *somefolder*: different encryption passwords used`. I am not saying the error messages are ideal, but I don't get where "report each fault separately" or any "long list" come into this.

Reply:
Right, ignore me, the specific error is in the message.